Library/Formula/multimarkdown.rb

```diff
@@ -30,9 +23,15 @@ def install
       system "make"
       bin.install "multimarkdown"
     end
+
+    bin.install Dir["scripts/*"].reject { |f| f =~ /\.bat$/ }
   end
 
   test do
+    Dir["scripts/*"].reject { |f| f =~ /\.bat$/ }.each do |unix_script|
```

Comment:
Shall I then hard-code the scripts to be copied in a class variable to make them available to the test method?